recipe/meta.yaml

```diff
@@ -1,31 +1,28 @@
 {% set name = "pydeck" %}
 {% set version = "0.7.1" %}
+{% set sha256 = "907601c99f7510e16d27d7cb62bfa145216d166a2b5c9c50cfe2b65b032ebd2e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 907601c99f7510e16d27d7cb62bfa145216d166a2b5c9c50cfe2b65b032ebd2e
+  sha256: {{ sha256 }}
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  # noarch: python
 
 requirements:
   host:
```

Comment:
Could you please add `jinja2` in `host`? See https://github.com/visgl/deck.gl/blob/6708cd139680863d17335600066257a2d16c2c68/bindings/pydeck/setup.py#L221

Reply:
I'm doing this now. Are you OK with me omitting the other setup dependencies, since we are using `pip --no-deps`?